src/rest_api/classes/gene/expression.clj

```diff
@@ -109,3 +110,324 @@
+     :life_stage (->> (:expr-pattern/life-stage expr-pattern)
+                      (map :expr-pattern.life-stage/life-stage)
+                      (map pack-obj)
+                      (seq))
+     :expressed_in (->> (:expr-pattern/anatomy-term expr-pattern)
+                        (map :expr-pattern.anatomy-term/anatomy-term)
+                        (map pack-obj)
+                        (seq))
+     :go_term (->> (:expr-pattern/go-term expr-pattern)
+                   (map :expr-pattern.go-term/go-term)
+                   (map pack-obj)
+                   (seq))
+     :database (seq (expr-pattern-db expr-pattern))}))
+
+(defn epic-expr-patterns [gene]
+  (let [db (d/entity-db gene)]
+    {:data
+     (let [pattern-dbids
+           (d/q '[:find [?ep ...]
+                  :in $ ?gene
+                  :where
+                  [?gh :expr-pattern.gene/gene ?gene]
+                  [?ep :expr-pattern/gene ?gh]
+                  [?ep :expr-pattern/epic]]
+                db (:db/id gene))]
+       (seq (map #(epic-table-row db %) pattern-dbids)))
+     :description "Large-scale cellular resolution compendium of gene expression dynamics"}))
+
+
+;; example gene with expression movies WBGene00016948
+(defn expression-movies [gene]
+  (let [db (d/entity-db gene)]
+    {:data
+     (let [expr-pattern-dbids
+           (d/q '[:find [?ep ...]
+                  :in $ ?gene
+                  :where
+                  [?gh :expr-pattern.gene/gene ?gene]
+                  [?ep :expr-pattern/gene ?gh]
+                  [?ep :expr-pattern/movieurl ?th]]
+                db (:db/id gene))]
+       (if (not-empty expr-pattern-dbids)
+         (reduce (fn [coll expr-pattern-dbid]
+                   (let [expr-pattern (d/entity db expr-pattern-dbid)]
+                     (assoc coll
+                            (:expr-pattern/id expr-pattern)
+                            {:object (pack-obj expr-pattern)
+                             :movie (:expr-pattern/movieurl expr-pattern)
+                             :details (str/join "; " (:expr-pattern/pattern expr-pattern))})))
+                 {} expr-pattern-dbids)))
+     :description "the tissue that the gene is expressed in"}))
+
+
+(defn expression-cluster [gene]
+  (let [db (d/entity-db gene)]
+    {:data
+     (let [expr-clusters
+           (d/q '[:find [?ec ...]
+                  :in $ ?gene
+                  :where
+                  [?h :expression-cluster.gene/gene ?gene]
+                  [?ec :expression-cluster/gene ?h]]
+                db (:db/id gene))]
+       (->> expr-clusters
+            (map (partial d/entity db))
+            (map #(assoc {}
+                         :expression_cluster (pack-obj %)
+                         :description (:expression-cluster/description %)))
+            (seq)))
+     :description "expression cluster data"}))
+
+
+;;
+;; anatomy functions field
+;;
+
+(defn- anatomy-functions-table-row [db [anatomy-function-dbids involved-dbid]]
```

Comment:
s/`antanomy-function-dbids`/`antanomy-function-dbid`/ ? i.e de-pluralise (looks like it's a single id since it's passed to `d/entity`)